cdap-docs/faq/source/cdap-startup.rst

```diff
@@ -0,0 +1,226 @@
+.. meta::
+    :author: Cask Data, Inc.
+    :copyright: Copyright © 2015 Cask Data, Inc.
+
+:titles-only-toc: true
+
+.. _faq-installation-startup:
+
+======================
+CDAP FAQ: CDAP Startup
+======================
+
+.. contents::
+   :depth: 2
+   :local:
+   :backlinks: entry
+   :class: faq
+
+
+Startup
+=======
+
+My CDAP install on CDH using Cloudera Manager doesn't startup |---| what do I do?
+---------------------------------------------------------------------------------
+We have a :ref:`tutorial <step-by-step-cloudera-add-service>` with instructions on how to install CDAP on CDH 
+(`Cloudera Data Hub <http://www.cloudera.com/content/www/en-us/resources/datasheet/cdh-datasheet.html>`__) 
+using `Cloudera Manager <http://www.cloudera.com/content/www/en-us/products/cloudera-manager.html>`__. 
+
+If, when you try to start services, you receive an error in ``stderr`` such as::
+       
+  Error found before invoking supervisord: No parcel provided required tags: set([u'cdap'])
+
+The error message suggests that you have not completed the last step of installing a
```

Comment:
No, it means there's no parcel that matches. Be explicit. It *suggests* that you didn't follow the instructions, but it *means* that the parcel isn't available, for whatever reason.